fetchmail/fetchmail.py

```diff
@@ -45,7 +44,8 @@ def run(connection, cursor):
     for line in cursor.fetchall():
         fetchmailrc = ""
         user_email, protocol, host, port, tls, username, password = line
-        options = "options ssl" if tls else ""
+        options = "options fetchall antispam 501, 504, 550, 553, 554"
```

Comment:
I disagree with fetchall here. I suspect you will get duplicate messages, don't you?